"Save to head" button not working

I noticed the "save to head" button in twix (manual calibration) as not working correctly for me. 
Whenever pressing it, the parameters are not saved and the error `[ERROR] server error: no such path: hardware_ids` is printed to the error log.
I was using the replayer, not an actual Nao.

Confirm that this Bug actually appears on actual Naos or newly recorded replays.
This feature is saves a lot of time on competitions so fixing it is very important.

--- a/crates/code_generation/src/execution.rs
+++ b/crates/code_generation/src/execution.rs
@@ -156,6 +156,9 @@ pub fn generate_replayer_struct(cyclers: &Cyclers, with_communication: bool) -> 
                                     let (parameters_subscriptions, _) = buffered_watch::channel(Default::default());
                                     communication_server.expose_source("parameters", parameters_receiver, parameters_subscriptions)?;
                                     communication_server.expose_sink("parameters", parameters_sender)?;
+                                    let (_, ids_receiver) = buffered_watch::channel((std::time::SystemTime::now(), hardware_ids));
+                                    let (ids_subscriptions, _) = buffered_watch::channel(Default::default());
+                                    communication_server.expose_source("hardware_ids", ids_receiver, ids_subscriptions)?;
                                     communication_server.serve(addresses, keep_running).await?;
                                     Ok(())
                                 })

